python/lsst/meas/algorithms/convertRefcatManager.py

```diff
@@ -454,3 +454,35 @@ def gaiaFluxToFlux(flux, zeroPoint):
         result['phot_rp_mean_fluxErr'] = result['phot_rp_mean_flux'] / input['phot_rp_mean_flux_over_error']
 
         return result
+
+class ConvertGaiaXpManager(ConvertRefcatManager):
+    """Special-case convert manager to deal with refcats that have no MagErrors
+    """
+    def _getFluxes(self, inputData):
+        """Compute the flux fields that will go into the output catalog.
+
+        Parameters
+        ----------
+        inputData : `numpy.ndarray`
+            The input data to compute fluxes for.
+            from Gaia Collaboration, Montegriffo et al. 2022 flux units are  W/Hz/(m^2) 
+
+        Returns
+        -------
+        fluxes : `dict` [`str`, `numpy.ndarray`]
+            The values that will go into the flux and fluxErr fields in the
+            output catalog, units are nJy
+        """
+        result = {}
+        for item in self.config.mag_column_list:
+            
+            inFluxKey=item.replace('_','_flux_')
+            outFluxKey=item + '_flux'
+            
+            inFluxErrKey=item.replace('_','_flux_error_')
```

Comment:
What is this for? The `mag_column_list` should have the actual name of the input fields, I would think. What are the names of the input flux fields in the xpSpec files?

Reply:
Example xp photometry column names are:
 'Lsst_flux_u',
 'Lsst_flux_error_u'
 I had thought we would want output to be:
 'Lsst_u_flux' or 'Lsst_u_fluxErr'
 But instead we want to preserve the original column name the fluxes come from in the xp catalog so output will be 
 'Lsst_flux_u_flux' or 'Lsst_flux_u_fluxErr'